src/widgets/Header/style.module.scss

```diff
@@ -100,16 +100,77 @@
 }
 
 .mobileHeader {
-  padding: rem(24px);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 80px;
+  padding: 0 rem(16px);
+
+  a {
+    display: flex;
+  }
 }
 
 .mobileNav {
   position: absolute;
-  width: 100vw;
-  height: 100vh;
-  z-index: 1000;
-  background-color: #0071f9; // TODO: Change color to background image.
   top: 0;
   left: 0;
+  width: 100vw;
+  height: 100dvh;
+  z-index: 1000;
+  background-color: #0071f9;
+  background-image: url('../../assets/images/textures/noise2.png'),
+    url('../../assets/images/gradient2.png');
```

Comment:
If you look at the CSS for the footer, the blue BG image and the noise image are in separate divs so that we can use `mix-blend-mode: overlay;` for the noise image. This will make the overall effect match the designs.